website.js

```diff
@@ -319,6 +316,17 @@ io.on('connection', function (socket) {
             socket.emit('nextSong');
         }})
     })
+    socket.on('createPoll', function (data) {
+        connection.query('insert into poll set ?', data, function(err, result) {})
+    })
+    socket.on('getPoll', function (data) {
+        connection.query('select * from poll where id = ?', data, function(err, result) {
+            socket.emit('pollData', result[0].answers)
+        })
+    })
+    socket.on('addResult', function (data) {
+        connection.query('update poll set answers = ? where id = "' + data.id + '"', JSON.stringify(data.answers), function(err,result){})
```

Comment:
This is a really bad practice. Stringifying an object to string and then unpacking is not the right way to solve this. You should take a look into joined tables. 

Think about it like this:

#### Table 1: Orders
| id | amount | customer_id |
|---|---------|-------------|
| 1  | $500 | 69 |
| 2 | $250 | 2 |
| 3 | $10 | 69 |

#### Table 2: Customers
| id | name | email |
|---|-------|------|
| 1 | Bob     | bobross@gmail.com |
| 69 | Henk | henk@gmail.com |

The tables now only reference to each other using id's, so no nested things. If you want to get all orders of a certain customer, you can do the following query:

```sql
SELECT * FROM orders WHERE customer_id = 69;
```

You would then receive back a list of the two orders placed by customer with id `69`.

If you want all orders by all customers, you can do the following (haven't tested, but I think it works):

```sql
SELECT * FROM customers INNER JOIN orders ON customers.id = orders.customer_id;
```

More info here:
- http://www.sql-join.com/
- https://www.codecademy.com/learn/learn-sql (See last exercise)
- https://www.youtube.com/watch?v=G1OLrfjHDyw

Feel free to ask if you have any questions.